### PR TITLE
Miniconda 4.12.0

### PR DESCRIPTION
**Pull request by pseudoyim**
Update links and add hashes for Miniconda v4.12.0.

### CODE REVIEW

File: docs/source/miniconda.rst [hunk 1]
```diff
@@ -36,15 +36,15 @@ Latest Miniconda Installer Links
...
+   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe>`_,``4fb64e6c9c28b88beab16994bfba4829110ea3145baa60bda5344174ab65d462``
+   MacOSX,`Miniconda3 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,``007bae6f18dc7b6f2ca6209b5a0c9bd2f283154152f82becf787aac709a51633``
+   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,``cb56184637711685b08f6eba9532cef6985ed7007b38e789613d5dd3f94ccc6b``
+   ,`Miniconda3 macOS Apple M1 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh>`_,``c788378ba4bfe75bd1cdebe127b0bce89a0c4dc44310b24d9e7fb2b730206586``
+   Linux,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_,``78f39f9bae971ec1ae7969f0516017f2413f17796670f7040725dd83fcff5689``
+   ,`Miniconda3 Linux-aarch64 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-aarch64.sh>`_,``5f4f865812101fdc747cea5b820806f678bb50fe0a61f19dc8aa369c52c4e513``
```
Comment (kathatherine) [inline docs/source/miniconda.rst:45]: So are the hashes on the latest installer not supposed to match the very first installers in the miniconda_hashes list? Like the hash for Linux-aarch64 is 5f4f865812101fdc747cea5b820806f678bb50fe0a61f19dc8aa369c52c4e513 here, but it's 47affd9577889f80197aadbdf1198b04a41528421aaf0ec1f28b04a50b9f3ab8 in miniconda_hashes. Still different installers?

Comment (kathatherine) [reply docs/source/miniconda.rst:45]: Oh wait, I found the same hash a bit further down the list with the 3.9 installers. I see now that that other hash was for 3.7.

**Review by kathatherine — APPROVED** [review]
LGTM

File: docs/source/miniconda.rst [hunk 2]
```diff
@@ -36,15 +36,15 @@ Latest Miniconda Installer Links
...
+   Windows,`Miniconda3 Windows 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86_64.exe>`_,``1acbc2e8277ddd54a5f724896c7edee112d068529588d944702966c867e7e9cc``
+   ,`Miniconda3 Windows 32-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Windows-x86.exe>`_,``4fb64e6c9c28b88beab16994bfba4829110ea3145baa60bda5344174ab65d462``
+   MacOSX,`Miniconda3 MacOSX 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh>`_,``007bae6f18dc7b6f2ca6209b5a0c9bd2f283154152f82becf787aac709a51633``
+   ,`Miniconda3 MacOSX 64-bit pkg <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.pkg>`_,``cb56184637711685b08f6eba9532cef6985ed7007b38e789613d5dd3f94ccc6b``
+   ,`Miniconda3 macOS Apple M1 64-bit bash <https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-arm64.sh>`_,``c788378ba4bfe75bd1cdebe127b0bce89a0c4dc44310b24d9e7fb2b730206586``
+   Linux,`Miniconda3 Linux 64-bit <https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh>`_,``78f39f9bae971ec1ae7969f0516017f2413f17796670f7040725dd83fcff5689``
```
Comment (csoja) [inline docs/source/miniconda.rst:44]: I'm not seeing the latest .pkg link for macOSX-arm64 - only the .sh link. Other than that, this all LGTM.

**Review by csoja — CHANGES_REQUESTED** [review]

File: docs/source/miniconda.rst (hunk 2 above)
Comment (csoja) [reply docs/source/miniconda.rst:45]: Thanks for adding it!

**Review by csoja — APPROVED** [review]
LGTM Thank you for updating the conda version for the latest as well!

**Review by skalburgi — APPROVED** [review]
LGTM

**Comment by jezdez** [comment]
@conda-bot check

**Review by jezdez — APPROVED** [review]

**Comment by jezdez** [comment]
Fixes #775.

**Pull request merged**